src/views/rewards/rewards.vue

```diff
@@ -15,12 +37,32 @@
 import { Vue } from 'vue-property-decorator';
 import Component from 'vue-class-component';
 import Header from '@/components/header/header-component.vue';
+import ProgressBannerComponent from '@/components/progress-banner/progress-banner-component.vue';
+import RewardDetails from '@/components/rewards-details/reward-details.vue';
+import { VenuesService } from '@/services/venue-service';
+import { log } from 'util';
```

Comment:
need dis?

Reply:
sure don't. I think this was imported as a result of one of my attempts to `console.log`